src/constants/localStorage.ts

```diff
@@ -1,4 +1,5 @@
 export enum LOCAL_STORAGE_KEYS {
+  FrontmatterSections = "frontmatter-sections",
```

Comment:
nit: rename to homepage-frontmatter-sections for clarity

Reply:
sgtm!